src/notification-toast/index.ts

```diff
@@ -0,0 +1,2 @@
+export * from "./notification-toast";
```

Comment:
Can we rename this whole component to just `Toast` and be in the folder `src/toast`